src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -56,7 +64,24 @@ export class WdsDataTableProvider implements DataTableProvider {
     supportsTypeRenaming: false,
     supportsExport: false,
     supportsPointCorrection: false,
-    supportsFiltering: false
+    supportsFiltering: false,
+    supportsTabBar: false
+  }
+
+  tsvFeatures: TSVFeatures = {
+    needsTypeInput: true,
+    uploadInstructions: 'Choose the data to import below. ',
+    sampleTSVLink: 'src/../wds_template.tsv', //TODO: placeholder, does not currently work
+    invalidFormatWarning: 'Invalid format: Data does not include sys_name column.',
+    isInvalid: (_0: boolean, filePresent: boolean, _2: boolean, sysNamePresent: boolean) => {
```

Comment:
Just a mini-nitpick, but perhaps a mini-docstring could be useful here for the `_0` and `_2` variables to state why they are not being used could be useful for readability.